core/rootblockchain.go

```diff
@@ -114,6 +120,29 @@ type RootBlockChain struct {
 	rootChainStakesFunc func(address account.Address, lastMinor common.Hash) (*big.Int, *account.Recipient, error)
 }
 
+// WriteStatus status of write
```

Comment:
What is the purpose of moving these functions in the same file?

Reply:
It is a format change, just want to move all the RootBlockChain func together.